Updated several images and descriptions for swate (#635)
nfdi4plants/nfdi4plants.knowledgebase: src/content/docs/swate/term-search.mdx

Reviewer: This page must also explain settings to configure all term searches in swate to use default search, or any TIB cataloge. Ask me if you need information on this

</TabItem>
</Tabs>

> *You can open the Swate term search details modal by clicking F2 on a term in the search. It will show additional information on its information tab. When you have swapped the tab in the modal, you can view the details again by clicking on the Details tab.*

Reviewer: make it <kbd>F2</kbd> (`<kbd>F2</kbd>`)

To execute the search query on any search text again after closing the dropdown with the search results, double click into the search field.
:::

## Adapting the Term Search

In the settings you can configure your term search in the "Term Search Configuration" part further. You can decide whether the default term search shall be used or not. Additionally, you can add and remove high performance search TIB terms to optimize your search further.

> ##### Default Search

Reviewer: ###*

> By using the default search, users also query the [DPBO](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/fundamentals/ontology-service-landscape/#how-can-i-find-a-certain-ontology-term-for-my-data-annotation), an ontology repository that is quickly updated based on user feedback. That way new terms are often added faster here than in other ontologies and enable the usage of these new term.

Reviewer: ```suggestion
Default search enables/disables search of the [SwateDB](/fundamentals/ontology-service-landscape/#swatedb-and-swobup). A DataPLANT maintained ontology database, which features a set of external ontologies, as well as our DPBO broker ontology, allowing us to quickly update and extend the terms offered via this search option.
```

Remember to verify correct link. And do not use absolute links to live service, instead use relative links.

> ##### TIB Terms

Reviewer: ```suggestion
### TIB Term Search
```

> The use of [TIB terms](https://terminology.tib.eu/ts/collections?col=DataPLANT) enables high-performance search capabilities. Through this optimized single access point, users can explore ontologies across a wide range of scientific fields besides those provided by DataPlant.

Reviewer: ```suggestion
The TIB database is a optimized for fast search speed and features different collections of ontologies. Through this optimized single access point, users can access different ontologies across a wide range of scientific fields, such as engineering or chemistry. 
```
